lighthouse-core/runner.js

```diff
@@ -66,9 +66,10 @@ class Runner {
       // Now run the audits.
       run = run.then(artifacts => Promise.all(config.audits.map(audit => {
         const status = `Evaluating: ${audit.meta.description}`;
-        log.log('status', status);
-        return Promise.resolve(audit.audit(artifacts)).then(ret => {
-          log.log('statusEnd', status);
+        return Promise.resolve().then(_ => {
+          log.log('status', status);
+          const ret = audit.audit(artifacts);
+          log.verbose('statusEnd', status);
```

Comment:
I think the way you had it before was more correct, since some of the audits do return a promise (and more will need to as we add derived artifacts), so `statusEnd` may be logged before those complete